config/routes.rb

```diff
@@ -59,6 +60,8 @@
     get 'games' => 'api#games'
     get 'dashboard_data' => 'api#dashboard'
     get 'dashboard_data/:game_id' => 'api#dashboard'
+    get 'mobile_dashboard_data' => 'api#mobile_dashboard'
```

Comment:
I guess...you're expecting users to just hit `localhost:3000/mobile_dashboard_data` to see the mobile version?

This is fine for now I guess but what we should really do is update games_controller#dashboard so that if it detects mobile, use a mobile-friendly rendering template instead of the existing one.  I can set that up later but please keep in mind that you should have the same vars set for both mobile and non-mobile version.